Makefile

```diff
@@ -0,0 +1,90 @@
+ENVIRONMENT?=staging
+DEPLOY_TARGET?=heroku
+
+# Associate the current time with the release.
+# This gets injected into the frontend and is used by Sentry
+VERSION=$(shell date +%Y-%m-%dT%H:%M:%S%z)
+
+HEROKU_APP_staging:=call-home-staging
+HEROKU_APP_prod:=call-home
+HEROKU_APP_TARGET:=$(HEROKU_APP_$(ENVIRONMENT))
+
+PUBLIC_URL_staging=https://call-home-staging.herokuapp.com
+PUBLIC_URL_prod=https://app.callhome.sg
+PUBLIC_URL_TARGET:=$(PUBLIC_URL_$(ENVIRONMENT))
+
+CLOUDRUN_REPO:=gcr.io/call-home-320615/call-home
+
+APPS:=shared frontend backend
+
+.DEFAULT_GOAL:=help
+
+##@ Dev
+
+init: $(APPS:%=%.init) ## Setup repo, and npm ci
+
+$(APPS:%=%.init): %.init:
+	cd $* && npm run init && npm ci
```

Comment:
Since `shared/` is a shared library, there's a dependency on `shared.init` in both `backend.init` and `frontend.init`, but these are peers in this rule. Technically not a problem if not running with concurrency, though not ideal. Maybe leave a TODO comment and we can come back to this later.

Reply:
done